rpc/src/services/base_services.rs

```diff
@@ -146,8 +166,15 @@ pub(crate) async fn get_block_header(
 }
 
 /// Get information about block shell header
+#[cached(
+    name = "BLOCK_SHELL_HEADER_CACHE",
+    type = "TimedSizedCache<(ChainId, BlockHash), Option<BlockHeaderShellInfo>>",
```

Comment:
@bkontur same situation here regarding the `Option` value, right?

Reply:
@tizoc 
uff, thx, I missed that